Header header

int32[] Mode
int32[] Operation
int32[] CmdValue
